.github/workflows/dependabot.yaml

```diff
@@ -0,0 +1,91 @@
+name: dependabot
+
+on:
+  pull_request:
+    types:
+      - opened
+  push:
+    branches:
+      - main
+
+permissions:
+  contents: read
+
+# Only run one instance per PR to ensure in-order execution.
+concurrency: pr-${{ github.ref }}
+
+jobs:
+  # Dependabot is annoying, but this makes it a bit less so.
+  dependabot-automerge:
+    runs-on: ubuntu-latest
+    if: github.event_name == 'pull_request' && github.event.pull_request.user.login == 'dependabot[bot]' && github.repository == 'coder/coder'
```

Comment:
I still think we should match on the ID here instead of the name.

Reply:
I am not sure if that will work. But I can try. 
This is from GitHub docs where they recommend using this context for doing this workflow.